stable/mysql/templates/configmap.yaml

```diff
@@ -2,10 +2,22 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "mysql.fullname" . }}
+  name: {{ template "mysql.fullname" . }}-configuration
 data:
 {{- range $key, $val := .Values.configurationFiles }}
   {{ $key }}: |-
 {{ $val | indent 4}}
 {{- end }}
+{{- end -}}
+---
```

Comment:
Please create a separate file for this.

Reply:
Hi @unguiculus,
thanks for your input.

- Done splitting configmaps into seperate files.
- Synced my fork
- rebased branch onto master

Puzzled however, why pull-charts-e2e is failing.